contrib/inventory/windows_azure.py

```diff
@@ -195,9 +195,7 @@ def add_cloud_services(self):
             for cloud_service in self.sms.list_hosted_services():
                 self.add_deployments(cloud_service)
         except WindowsAzureError as e:
-            print("Looks like Azure's API is down:")
-            print("")
-            print(e)
+            prin("Looks like Azure's API is down:")            print(e)
```

Comment:
This line is generating syntax errors.  `prin` should likely be `print` and the second `print` should be on it's own line.